package-examples/wordpress-in-a-box/mysql/deployment.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: PROJECT_ID-yourspace # {"$kpt-set":"proj-namespace"}
```

Comment:
Why are we not using the `default` namespace as the default here?

Reply:
It is an example where package publisher marks namespace as a required setter and consumer must set it before using the blueprints.